src/mpcCoreKit.ts

```diff
@@ -312,20 +314,6 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
       await this.featureRequest();
       TorusUtils.setSessionTime(this.options.sessionTime);
     }
-
-    // if not redirect flow or session rehydration, ask for factor key to login
-
-    // dont wait for wasm to be loaded, we can reload it during signing if not loaded
```

Comment:
why do we skip async pre-load?
this can reduce the time needed for the first signing

Reply:
matthias said, its just 30ms for loading wasm lib so i thought its fine to remove it for the sake of code clarity